### ISSUE TITLE
Screen reader, orca is very badly integrated to Cinnamon

### ISSUE THREAD

**Issue by ksb749** [linuxmint/cinnamon#12401]
### Distribution

Debian 12.7 Cinnamon DE

### Package version

5.6.8

### Graphics hardware in use

Nvidia Geforce 1060

### Frequency

Always

### Bug description

Im blind, and use Orca screen reader, and wish to use Cinnamon desktop enovirement, but Orca is not able to read the elements in Cinnamon menu, either if i type text in the search field, but the search does give results.. As im new and not use Cinnamon as blind, i dont know where to start this issue, but i hope you will help making Cinnamon useable for blind people using Orca screen reader.

### Steps to reproduce

SImply install Debian 12.7 with cinnamon and enable Orca screen reader, then you will find out how inaccessible it is.

### Expected behavior

It should be possible for blind users to use Cinnamon desktop enoviremnt as everybody else with Orca screen reader
